src/shogun/base/SGObject.h

```diff
@@ -951,6 +967,16 @@ class CSGObject
 	Unique<ParameterObserverList> param_obs_list;
 
 protected:
+
+	/**
+	 * Return total subscriptions
+	 * @return total number of subscriptions
+	 */
+	index_t get_total_subscriptions() const
```

Comment:
`get_num_subscriptions` -- to be consistent with the string registered in "watch_method"